shared/constant/constant.go

```diff
@@ -16,10 +16,11 @@ package constant
 
 // PrivilegedPlugins can be changed by 'WOODPECKER_ESCALATE' at runtime
 var PrivilegedPlugins = []string{
-	"plugins/docker",
-	"plugins/gcr",
-	"plugins/ecr",
-	"woodpeckerci/plugin-docker-buildx",
+	"docker.io/plugins/docker",
+	"docker.io/plugins/gcr",
+	"docker.io/plugins/ecr",
+	"docker.io/woodpeckerci/plugin-docker-buildx",
+	"quay.io/woodpeckerci/plugin-docker-buildx",
```

Comment:
maybe add also codeberg?
```suggestion
	"quay.io/woodpeckerci/plugin-docker-buildx",
	"codeberg.org/woodpecker-plugins/docker-buildx",
```